src/app/src/test/kotlin/org/vaccineimpact/api/tests/security/WebTokenHelperTests.kt

```diff
@@ -62,6 +62,22 @@ class WebTokenHelperTests : MontaguTests()
         assertThat(claims["permissions"]).isEqualTo("*/p1,prefix:id/p2")
     }
 
+
+    @Test
+    fun `can generate shiny token for report reviewer`()
+    {
+        val roles = listOf(
+                ReifiedRole("report.reviewer", Scope.Global())
+        )
+        val token = sut.generateShinyToken(MontaguUser(properties, roles, permissions))
+        val claims = sut.verify(token)
+
+        assertThat(claims["iss"]).isEqualTo("vaccineimpact.org")
+        assertThat(claims["sub"]).isEqualTo("test.user")
+        assertThat(claims["exp"]).isInstanceOf(Date::class.java)
+        assertThat(claims["allowed_shiny"]).isEqualTo(true)
+    }
```

Comment:
I feel like the better place for checking that it's true when you're a report-reviewer and false otherwise is here, in the unit test, rather than the blackbox test.

Reply:
You can leave the blackbox test if you like, now you've written it - although on the other hand, we want to keep the time spent on integration tests down, so maybe not?

Reply:
I've removed blackbox test

Reply:
and added one here as suggested, you're totally right